lib/client.ml

```diff
@@ -72,15 +72,23 @@ end
 
 module Rpc = Mux.Make(NbdRpc)
 
-type info = {
-  read_write: bool;
-  sector_size: int;
-  size_sectors: int64;
-}
+(* samoht: `Msg should be the list of all possible exceptions *)
+type error = [ Mirage_block.error | `Msg of string ]
```

Comment:
Ideally the `Msg of string` should be a more concrete error, e.g. something like `Protocol of Protocol.Error.t` or similar, and the `pp_error` will do the conversion to a string.